src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,63 @@
+@keyframes moveHand {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
You can make a separate file for animations, it will be much easier in the future to find the required animation if it is used in many blocks.